packages/augur-simplified/src/utils/date-utils.ts

```diff
@@ -30,3 +30,19 @@ export const getTimeFormat = (timestamp) => {
   const minutes = `0${Number(date.getMinutes())}`.slice(-2);
   return `${hours}:${minutes}`
 }
+
+export const getMarketEndtimeDate = (timestamp: string | number) => {
+  const inMilli = Number(timestamp) * 1000;
+  const date = new Date(inMilli)
+  const year = date.getFullYear();
+  const monthDay = getDayFormat(timestamp);
+  return `${monthDay} ${year}`
+}
+
+export const getMarketEndtimeFull = (timestamp: string | number) => {
+  if (!timestamp) return 'Missing';
+  // use existing to make sure to be consistent
+  const monthDay = getMarketEndtimeDate(timestamp);
```

Comment:
Variable is named `monthDay` yet it represents `monthDay and year`

Reply:
updated